docs/contributing/pull-requests/branching.md

```diff
@@ -0,0 +1,246 @@
+Feature Branch. The reviewer can be anyone in the pod. However, since the work this branch is
+typically larger, some seniority with the codebase can be beneficial.
+
+#### QA
+
+The QA team tests on the Long-Lived Feature Branch (note that QA comes **after** the PR is
+completed). This is because the premise of this flow is that the individual parts of the feature
+cannot be tested individually.
+
+If QA finds a defect, it should be fixed in another Issue Branch off of the Long-Lived Feature
+Branch, with another reviewed PR into the Long-Lived Feature Branch to address the problem.
+
+#### Reviewing
+
+When all development and functional testing is complete on the Feature Branch, the original PR into
+`master` should be moved out of Draft status and tagged with the appropriate development group for
+review.
+
+We can perform the review using GitHub's UI. By opening the Pull Request and clicking on the
+`Commits` tab. Afterwards check each commit individually.
+
+- Verify the commit has an existing review by following the Pull Request link and verifying the
+  commit SHA hash matches. `Author merged commit 8c948fd into branch`.
+- If not perform a regular code review of the commit.
+
+Merge commits should be reviewed as well, the GitHub UI will automatically simplify the merge commit
+and only show the changes made. If reviewing from the command line or through a different tool,
+please use the command `git show <hash>`. For some background and more information please read
+[How to review a merge commit](https://haacked.com/archive/2014/02/21/reviewing-merge-commits/).
+
+### Short-Lived Feature Branch
+
+A Short-Lived Feature Branch is a good fit for bodies of work that:
+
+- Can be developed by a single contributor
+- Can be reviewed in a single Pull Request
+- Can be tested independently, and
+- Can be released independently
+
+This will often be the case for small pieces of new functionality and for most bug fixes.
+
+#### Development
+
+The developer should create a branch named with the initiating Jira issue (e.g. `sg/SG-123`) and
+create a **draft** PR from that branch into `master`.
+
+:::note
+
+The branch name should be as short as possible, preferably just the issue name. This makes it easier
+for the QA team to switch environments to individual branches, as they have to type in the branch
+name multiple times when doing so. This is especially important for Short-Lived Feature Branches, in
+which testing may be much briefer.
+
+:::
+
+During development, the `master` branch should be regularly merged into the branch to avoid
+conflicts.
+
+When development is complete, the developer should prepare the PR for review:
+
+- Remove the Draft status from the PR
+- Add the `needs-qa` label to the PR
+- Tag the appropriate development group for review
+
+**Unlike with a Long-Lived Feature Branch, when a team member approves the PR, it should remain open
+to be tested.**
```

Comment:
We seem to follow this for long lived branches as well. Remove?

Reply:
Do we?  On the SG pod, when an individual developer finishes their task that is part of a feature (I'll call this the "issue branch"), they complete the PR to merge that change into the feature branch, and QA tests on the feature branch.  This was our operating model as it allows QA to keep their environments pointed to one place instead of switching around.

I do see your point that if we were to test **before** merging it would avoid the mental math that you have to do every time you have a PR approved.  (i.e. "Is this merging into a feature branch, so I complete it?  Or is it merging into `master` so I leave it open for testing?")

Reply:
Ah now I get it. I read it as the long lived feature branch could be merged to master. Can we rephrase it to make it more clear that we mean merging into a feature branch, not merging the feature branch?

Reply:
I clarified this.  It was confusing and could be read in different ways.